sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultPlanManager.java

```diff
@@ -13,6 +13,7 @@
     private final Plan plan;
 
     public DefaultPlanManager(final Plan plan) {
+        plan.getStrategy().interrupt();
```

Comment:
Similar to `DefaultScheduler.java:L450` it'd make sense to add a comment explaining the logic here, possibly cross-referencing with the `proceed()` call there if the two are related..